### ISSUE TITLE
Welcome flow: remove the confusing faux "hubot" dialog

### ISSUE THREAD

**Issue by joewiz** [desktop/desktop#2141]
### Description

During the welcome flow, users are presented with this screen:

> ![screen_shot_2017-06-28_at_10_52_56_am](https://user-images.githubusercontent.com/59118/27644821-82479532-5bf2-11e7-8167-a0b05afc6272.png)

The little "hubot" dialog thing indicated by the pink arrows is quite likely to confuse users ("what is hubot?" "why is this 'Fix all the things' next to my name and avatar?") and provides no information or discernible benefit. Thus, I recommend that this image be removed or replaced.

### Version

**GitHub Desktop version:** 0.6.2

**OS version:** macOS 10.12.5

### Steps to Reproduce

1. Perform first run with GitHub Desktop (clearing `~/Library/Application Support/GitHub Desktop` to restore first run status)
2. Follow prompts to authenticate with GitHub.com. You'll see the image above.

**Expected behavior:** No confusing images during first run.

**Actual behavior:** Confusing images during first run

**Reproduces how often:** 100%

### Logs

n/a

#### Additional Information

n/a

**Comment by shiftkey** [comment]
@joewiz for context: we added this section so that you'd get a preview of what committing looks like in the app - to ensure you have entered an address that GitHub will recognize.

We received support requests from people using the classic apps who didn't see what they expected when they were making commits in the app, and this is intended to catch those situations from occurring.

**Comment by j-f1** [comment]
Why not say something like “✓ Associated with <img src="https://avatars1.githubusercontent.com/u/25517624?v=3&s=40" height="16" align="center"> @j-f1 on GitHub?”

**Comment by shiftkey** [comment]
@j-f1 because most of the issues we see are related to mismatches in email addresses and their avatar not rendering. We're tracking the email check in #610, but the visual cue is probably more noticeable at this point to users.

**Comment by j-f1** [comment]
Then it could say:
> ❌  Not associated with any GitHub account  
> ⚠ This means your commits won’t show an avatar, and they won’t count as contributions in your GitHub profile

**Comment by niik** [comment]
We talked briefly about this in the design review and we see three possible paths forward. We could remove the list entirely, or we could turn it into a single commit preview, something like

![image](https://user-images.githubusercontent.com/634063/28082363-94f40398-6672-11e7-988d-30fbb78005ea.png)

We haven't discussed the warning approach outlined above yet. We'll need to think about this some more. We'll try to revisit next week.

**Comment by joshaber** [comment]
@desktop/design decided to move forward with the above mock ☝️

**Comment by niik** [comment]
> @desktop/design decided to move forward with the above mock ☝️

We did indeed but I'd like to add that we don't consider this to be the end this of discussion. We've got some ideas around this particular part of the welcome flow that's not quite hashed out yet. What we decided on, however, is that the mock seen above is better than what we have today and since we have limited bandwidth we decided to go with what we've got for 1.0 and revisit later.
